### DISCUSSION TITLE
How to set up with HTML code?

### DISCUSSION THREAD

**Discussion by JurassicNiffler** [EmulatorJS/EmulatorJS#371] [Q&A]
I am new to git hub but I have been using emulatorjs on another website and I don't know how this works, but do you have a html code that emulates it?

**Comment by ethanaobrien** [comment]
Well technically it's C/C++ code compiled to webassembly that runs it, with a JavaScript frontend, but I'm assuming you're just asking for the instructions to set it up, which would be this tool - https://emulatorjs.netlify.app/codehelper

**Reply by JurassicNiffler** [reply]
thanks that helps but the link is not working for some reason

**Reply by ethanaobrien** [reply]
Very sorry, try this one: https://emulatorjs.netlify.app/editor.html

**Reply by JurassicNiffler** [reply]
thank you I'm trying to emulate this file because I'm making a unblocked games website for my school I'm not sure how to do it right but I try and figure it out. Its for GBA

Edited to remove rom

**Reply by allancoding** [reply]
Please remove the link the the rom, if you want to give us a rom to test please email it.

**Reply by ethanaobrien** [reply]
I don’t want to deal with the roms, I try to keep away from that since it’s technically illegal. I always use homebrew roms for testing so please contact @allancoding for help if it includes sending a rom

**Reply by JurassicNiffler** [reply]
sorry I didnt know
